eth/main.cpp

```diff
@@ -962,10 +959,7 @@ int main(int argc, char** argv)
 	for (auto const& s: passwordsToNote)
 		keyManager.notePassword(s);
 
-	writeFile(configFile, rlpList(signingKey, author));
-
-	if (sessionKey)
-		signingKey = sessionKey;
+	writeFile(configFile, rlpList(author, author));
```

Comment:
Not sure about this, probably we can't change the format of this file easily? Or do we need it at all